openapi/connector_mgmt-private-admin.yaml

```diff
@@ -765,6 +765,65 @@ paths:
         - Bearer: [ ]
       operationId: getConnector
       summary: Get a connector
+
+    patch:
+      tags:
+        - Connector Clusters Admin
+      security:
+        - Bearer: [ ]
+      operationId: patchConnector
+      summary: Patch a connector
+      description: Patch a connector
+      requestBody:
+        description: Data to patch the connector with
+        content:
+          application/merge-patch+json:
+            schema:
+              type: object
+          application/json-patch+json:
+            schema:
+              description: A JSON Patch, RFC 6902 - https://tools.ietf.org/html/rfc6902
+              type: object
+          application/json:
+            schema:
+              $ref: "connector_mgmt.yaml#/components/schemas/ConnectorRequest"
```

Comment:
The admin API follows the pattern that if there is a request/response type that's specific to the admin API. It uses a type defined in this file with the naming convention like `*AdminView`. 
Since the admin patch API is only allowed to patch the state, the admin API should use a type ConnectorAdminRequest, which only has properties that are allowed to be changed by the admin API. Otherwise the API makes it seem like even the connector configuration can be patched.

Reply:
Fixed with https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1365/commits/59704582f0cc293bba388ecfd4a4a760ae46114d